DEVELOPMENT.md

```diff
@@ -74,28 +74,45 @@ When enabled by default, this also works with editor plugins like
 
 [vim-fugitive]: https://github.com/tpope/vim-fugitive
 
-### iBazel: A file watcher for Bazel.
+### iBazel and dev target
+
+To make the devleopment faster, we can use run TensorBoard on dev target with iBazel.
+
+```sh
+(tf)$ ibazel run tensorboard:dev -- \
+--logdir path/to/logs \
+[--bind_all] \
+[--port PORT_NUMBER]
+```
 
-Bazel is capable of performing incremental builds where it builds only the
-subset of files that are impacted by file changes. However, it does not come
-with a file watcher. For an improved developer experience, start TensorBoard
-with `ibazel` instead of `bazel` which will automatically re-build and start the
-server when files change.
+* `ibazel`: Bazel is capable of performing incremental builds where it builds only
```

Comment:
Nit: inconsistency in indentation. \`ibazel\` line has one less space than 'the subset' line which has one less space than the \`:dev\` line.